elasticsearch/example/src/test/java/example/springdata/elasticsearch/conference/ElasticsearchOperationsTest.java

```diff
@@ -103,4 +109,29 @@ void geoSpatialSearch() {
 
 		assertThat(result).hasSize(2);
 	}
-}
+
+    @Test
+    void shouldUpdateConferenceKeywords() {
+        var conferenceName = "JDD14 - Cracow";
+        var newKeyword = "java-ee";
+
+        var initialQuery = new CriteriaQuery(new Criteria("name").is(conferenceName));
+        SearchHit<Conference> searchHit = template.searchOne(initialQuery, Conference.class);
+        assertThat(searchHit).isNotNull();
+
+        Conference conference = searchHit.getContent();
+        String conferenceId = searchHit.getId();
+
+        int originalKeywordsCount = conference.getKeywords().size();
+        assertThat(conference.getKeywords()).doesNotContain(newKeyword);
```

Comment:
Same as above, the need for asserting initial state hints at a potential problem  with creating the state and distracts from the thing we actually want to explain here.